src/te/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -525,8 +542,13 @@ void InjectInline(ScheduleNode* sch, bool feature_extraction_mode) {
         for (auto iv : compute->axis) {
           args.push_back(iv->var);
         }
+        if (ext_ops.find(stage->op) != ext_ops.end()) {
+          // sshtin: The extern op can try to get access to the input tensors as a row data,
+          // that can lead to error in TE scripts.
+          stage->attach_type = kGroupRoot;
+          continue;
+        }
```

Comment:
So the new implementation of concat is implemented via IRBuilder, hence it is an `extern` op. The "op pattern" of an extern op is supposed to be `kOpaque` for fusion, but since the existing implementation is of kind `kInjective` and @shtinsa wants to preserve op fusion as before, he came up with the hack above.

thoughts? @tqchen